multiapi-engine/src/main/java/com/sngular/api/generator/plugin/commonApi/MapperUtilCommon.java

```diff
@@ -0,0 +1,199 @@
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public abstract class MapperUtilCommon {
+
+  public MapperUtilCommon() {}
+
+  public  static String getSimpleType(final JsonNode schema, final String prefix, final String suffix) {
+    String type;
+    if ((schema.has(TYPE)) && (!NULL.equalsIgnoreCase(schema.get(TYPE).toString()))){
+      type = schema.get(TYPE).textValue();
+      String format = null;
+      if (STRING.equalsIgnoreCase(type)) type = STRING;
+      if (schema.has(FORMAT)) format = schema.get(FORMAT).textValue();
+
+      if (NUMBER.equalsIgnoreCase(type)) {
+        if (FLOAT.equalsIgnoreCase(format)) {
+          type = FLOAT;
+        } else if (DOUBLE.equalsIgnoreCase(format)) {
+          type = DOUBLE;
+        } else {
+          type = BIG_DECIMAL;
+        }
+      } else if (INTEGER.equalsIgnoreCase(type)) {
+        if (INT_64.equalsIgnoreCase(format)) {
+          type = LONG;
+        } else {
+          type = INTEGER;
+        }
+      }
+
+    } else  {
+      type = getPojoName(getRefClass(schema), prefix, suffix);
+    }
+    return StringUtils.capitalize(type);
+  }
+
+  public static String getTextValue(JsonNode node, String field){
+    try{
+      if(checkTextValueNotNull(node.get(field)))
+        return node.get(field).textValue();
+    }
+    catch (StackOverflowError e){
+      return null;
+    }
+    return null;
+  }
+  public static boolean checkTextValueNotNull(JsonNode schema){
+    return (schema != null && schema.textValue()!= null && !schema.isNull());
+  }
+
+  public static String getPojoName(final String namePojo, final String prefix, final String suffix) {
+    return StringUtils.defaultIfBlank(prefix, "")
+            + StringUtils.capitalize(namePojo)
+            + StringUtils.defaultIfBlank(suffix, "");
+  }
+
+  public static String getRefClass(final JsonNode schema) {
+    final String[] pathObjectRef = schema.get(REF).textValue().split(SLASH);
+    return pathObjectRef[pathObjectRef.length - 1];
+  }
+
+  public static String getTypeMap(final JsonNode mapSchema, final String prefix, final String suffix) {
+    var typeMap = "";
+    if (mapSchema.get(ADDITIONAL_PROPERTIES)==null) return null;
+    final var mapNode = mapSchema.get(ADDITIONAL_PROPERTIES);
+    final var mapValueType = mapNode.findPath(TYPE);
+    typeMap = getCollectionType(mapNode, mapValueType, prefix, suffix);
+    return typeMap;
+  }
+
+  public static String getCollectionType(final JsonNode mapNode, final JsonNode mapValueType, final String prefix, final String suffix) {
+    String typeMap = null;
+    if ((checkTextValueNotNull(mapValueType))&&(!mapValueType.textValue().contains("#"))) {
```

Comment:
Why at the beginning of this method use var, and in the next one use String, checking other parts of the code you did the same, I think you should take a way to choose var or String to avoid problems with readability, because looks like each one with a different type of variable could be uses for different purpose